hellsemble/hellsemble.py

```diff
@@ -175,10 +176,12 @@ def __fit_estimators_sequential(
                 list of fitted estimators and the list of masks indicating
                 which observations were used during fit of the estimators.
         """
+        X_train, X_val, y_train, y_val = train_test_split(X, y, test_size = 0.25)
```

Comment:
We should introduce `seed` parameter to the `fit` method and pass it to every `train_test_split` to ensure reproducibility